content/guides/deployapp/appcli.md

```diff
@@ -0,0 +1,79 @@
++++
+title = "Deploy a New Application with CLI"
+weight = 300
+chapter = false
+pre = "2.1 "
++++
+{{% notice warning %}}
+**Prerequisites**
+
+1) Configure the [Core Platform CLI](../../corectl/).
+2) Complete the [Onboarding Tenancy](../onboarding/).
+{{% /notice %}}
```

Comment:
This is coming from this PR, and I agree that we should maintain a consistent style for prerequisites.

In my opinion, we should highlight prerequisites (e.g., using notices, warnings, or something similar) because, when there is too much information on a page, users tend to overlook some content. Therefore, I prefer using colors in prerequisites to capture the reader's attention.

Reply:
Sounds good - I'd vote for either a note block or an important block (warning feels slightly harsh!) 👍